docs/openrpc.json

```diff
@@ -0,0 +1,917 @@
+{
+  "openrpc": "1.3.2",
+  "info": {
+    "title": "Hathor wallet-to-dApp API",
+    "description": "API exposed by wallet applications for integration with dApps on Hathor platform.",
+    "contact": {
+      "name": "Hathor Labs",
+      "url": "https://docs.hathor.network/references/besides-documentation#support"
+    },
+    "license": {
+      "name": "MIT",
+      "url": "https://github.com/HathorNetwork/hathor-rpc-lib/blob/release/LICENSE"
+    },
+    "version": "0.1.0"
```

Comment:
Is this the package version? If it is, it should be `0.0.2-experimental-alpha`

Reply:
Yes. This is the package version.

I suggest that the package versioning be changed:

From:
0.0.2-experimental-alpha

To:
0.1.1-experimental-alpha

Or:
0.2.0-experimental-alpha

(depending on whether the change between them was a new feature or a bug fix).

Let me explain:

- Following the semantic versioning standard https://semver.org , the first version released, even if still in the development phase, should be 0.1.0 and from then on increment the minor version for each subsequent release.

- In other words: the first version should start counting with minor and not with patch.

- According to: https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase

Reply:
We can change it but in another PR. My suggestion is that we create a new PR to update to 0.1.0-experimental-alpha but this `docs.openrpc.json` should be the same as in the package.json. So I suggest you set as 0.0.1-experimental-alpha now and in the other PR we change it as well

Reply:
OK: change made:
- Version changed to "0.0.3-experimental-alpha", which is the latest one in master.